ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -774,8 +780,7 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
           </div>
         </WorkspaceEditSection>
         <WorkspaceEditSection header='Request a review of your research purpose for potential
-                                      stigmatization of research participants'
-                              tooltip={toolTipText.reviewRequest}>
+                                      stigmatization of research participants' required>
```

Comment:
I like the addition of `required` here. 

Not sure why, but I don't see  `(Required)` added after the header.